Formula/wget.rb

```diff
@@ -37,7 +38,6 @@ def install
                           "--sysconfdir=#{etc}",
                           "--with-ssl=openssl",
                           "--with-libssl-prefix=#{Formula["openssl@1.1"].opt_prefix}",
-                          "--disable-debug",
```

Comment:
`brew` doesn't support compiling with debug symbols, so removing this flag won't actually do anything.

Reply:
What do you mean? Removing the flag locally allowed me to do `wget --debug` whereas the bottled `wget` does not.

Reply:
Typically `--disable-debug` disables compiling with debug symbols by removing `-g` flags. `brew`'s compiler shims strips all `-g` flags, so no debug symbols are generated even when the build system tries to use them.

If it's changing what you see from `wget` then this flag may do something else, but if it still relies on debug symbols being there we probably don't want to remove this flag.

Reply:
MacPorts builds `wget` with `--enable-debug` so I think it is probably a feature flag rather than a compiler flag.

Reply:
From `./configure --help`:

```
  --disable-debug         disable support for debugging output
```